common/STcommon.py

```diff
@@ -1,10 +1,46 @@
 try:
 	import logging
+	import argparse
+	import configparser
 	from Crypto.PublicKey import RSA
+	from os.path import isfile
 except ImportError as e:
 	print("[-] {}, exiting".format(e))
 	exit(1)
 
+class Settings():
+	def __init__()
+		settings = configparser.ConfigParser()
+		settingsFile = ""
+		loaded = False
+
+	def createSettings(self,inFile='STcommon/settings.ini'):
+		self.settingsFile = inFile
```

Comment:
My original intention was to have the user call `loadSettings`, which would then call `createSettings` if needed. But then of course, it would make more sense to make `createSettings` private, which I mistakenly didn't do.